src/Client/Campaign.php

```diff
@@ -33,14 +33,12 @@
  *      instagram: ?string,
  *      optInStatement: ?string,
  *      stripeAccountId: string,
- *      hmrcReferenceNumber: string,
+ *      hmrcReferenceNumber: string|null,
  *      giftAidOnboardingStatus: string,
- *      regulator: string,
  *      regulatorRegion: string,
  *      regulatorNumber: string,
  *     },
  *     endDate: ?string,
- *     feePercentage: ?float,
```

Comment:
I added type to a campaign constant array in test so SA tools made me fix this. Was going to give up on Psalm's unreadable error message but then PHPStan told me what was wrong one line at a time.